grafana-plugin/src/plugin/PluginSetup/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, PropsWithChildren, useCallback, useEffect } from 'react';
```

Comment:
extracted this from `src/plugin/PluginConfigPage/PluginConfigPage.tsx`, this made testing this component much easier.